bpf/lib/trace.h

```diff
@@ -110,6 +126,21 @@ static inline void send_trace_notify(struct __sk_buff *skb, __u8 obs_point, __u3
 static inline void send_trace_notify(struct __sk_buff *skb, __u8 obs_point, __u32 src, __u32 dst,
 				     __u16 dst_id, __u32 ifindex, __u8 reason)
 {
+	switch (obs_point) {
+		case TRACE_TO_LXC:
+			update_metrics(skb->len, DIRECTION_INGRESS, REASON_FORWARDED);
+			break;
+
+		/* TRACE_FROM_LXC, i.e endpoint-to-endpoint delivery
+		 * is handled separately in ipv*_local_delivery() where we can bump
+		 * an egress forward. It could still be dropped but it would show
+		 * up later as an ingress drop, in that scenario.
+		 */
+		case TRACE_TO_HOST:
+		case TRACE_TO_STACK:
+		case TRACE_TO_OVERLAY:
+			update_metrics(skb->len, DIRECTION_EGRESS, REASON_FORWARDED);
+	}
```

Comment:
Same comment, because this code applies to both. Ideally we would not duplicate this code, but that's entirely cosmetic cleanup.

Reply:
I can do this as the followup PR. Thanks